### ISSUE TITLE
Review of Dopa-responsive dystonias

### ISSUE THREAD

**Issue by kanems** [monarch-initiative/mondo#5433]
I am trying to align MedGen concepts with Mondo/Orpha/OMIM/SNOMED (and eventually UMLS CUI mapping) and am running into some issues with names, which will probably take review by multiple groups to potentially sort out...

MONDO:0016812  dopa-responsive dystonia (DRD) is currently parent to:
Mondo ID | Mondo Name | OMIM Xref (name)| Orpha xREF (name) | SNOMED xREF(name) | Associated Gene
-- | -- | -- | -- | -- | --
MONDO:0011551 | TH-deficient DRD | 605407 (SEGAWA SYNDROME, AUTOSOMAL RECESSIVE) | 101150 (Autosomal recessive dopa-responsive dystonia, aka DYT5a) | 715827001 (Autosomal recessive dopa responsive dystonia) | TH  (Orphanet also has  TSPOAP1)
MONDO:0012994 | (DRD) due to sepiapterin reductase deficiency | 612716 (Dystonia, dopa-responsive, due to sepiapterin reductase deficiency) | 70594 (Dopa-responsive dystonia due to sepiapterin reductase deficiency) | 45116002 (Sepiapterin reductase deficiency) | SPR 
MONDO:0007495 | Dystonia 5 | 128230 (DYSTONIA, DOPA-RESPONSIVE)  | 98808 (Autosomal dominant dopa-responsive dystonia, aka DYT5b) | 715768000 (Autosomal dominant dopa responsive dystonia) | GCH1, GCH1 - AD or AR in OMIM; GCH1, IMPDH2 NR4A2 in Orpha*

Other OMIM records that may be relevant:
233910 INCLUDED entry:  DYSTONIA, DOPA-RESPONSIVE, WITH OR WITHOUT HYPERPHENYLALANINEMIA, AUTOSOMAL RECESSIVE, Gene on this record in OMIM is GCH1.

The SNOMED CT data, as represented in UMLS has:
230332007 “Diurnal dystonia” with three children:
-Autosomal Dominant (AD) DRD
-Autosomal recessive(AR) DRD
-DRD due to Sepiapterin reductase deficiency

So it looks like there are really 4 children concepts… Maybe?
I.	Dopa-responsive dystonia (aka Diurnal dystonia)
  a.	AD DRD due to GCH1 deficiency  (primary concept on OMIM 128230?)
  b.	AR DRD due to GCH1 deficiency (included concept on OMIM 233910?)
  c.	AR DRD due to TH deficiency (aka AR Segawa syndrome, OMIM 605407)
  d.	(DRD) due to sepiapterin reductase deficiency (AR, ?AD) (OMIM and OrphaIDs seem OK on this one)

@ahamosh Is OMIM Moving all the GCH1-associated DRDs to 128230 ? Right now the AR concept is 'included' on 233910,  but the Gene:Phenotype table on 128230 has both AD and AR inheritance modes. Would OMIM consider updating the preferred name on 128230 so that 'Dopa responsive dystonia' is a synonym since it seems there are at least 3 genes that contribute to DRD phenotypes?

I'm not sure where "DTY5a" and "b" come from and if DYT5 (dystonia 5) should really be a grouping term? OMIM links this to GCH1 associated DRD; but Orpha's "a" and "b" forms are associated with two different genes.
This article in the Genetics section of MedLinePlus suggests "Dystonia 5, dopa responsive type" describes all the DRDs, regardless of genetic cause: https://medlineplus.gov/genetics/condition/dopa-responsive-dystonia/

*side/final notes Regarding Orpha’s other genes associated with 98808:
- Gene NR4A2 (a nuclear receptor) is associated with OMIM phenotype 619911 (Dopa-responsive parkinsonism as a feature but also with intellectual disability)
-Gene IMP DEHYDROGENASE 2; IMPDH2 is associated with variants that affect enzyme activity but no DRD or movement disorder type condition seems to be specifically associated with this gene in OMIM.

**Comment by nicolevasilevsky** [comment]
> Other OMIM records that may be relevant:
233910 INCLUDED entry: DYSTONIA, DOPA-RESPONSIVE, WITH OR WITHOUT HYPERPHENYLALANINEMIA, AUTOSOMAL RECESSIVE, Gene on this record in OMIM is GCH1.

this is [GTP cyclohydrolase I deficiency with hyperphenylalaninemia](http://purl.obolibrary.org/obo/MONDO_0100186), which is a sibling of [dystonia 5](http://purl.obolibrary.org/obo/MONDO_0007495)

**Comment by nicolevasilevsky** [comment]
> Would OMIM consider updating the preferred name on 128230 so that 'Dopa responsive dystonia' is a synonym since it seems there are at least 3 genes that contribute to DRD phenotypes

OMIM:128230 is equiv to MONDO:0007495 Dystonia 5

**Comment by nicolevasilevsky** [comment]
@kanems I'm not sure I follow everything you said above. This is what we currently have in Mondo. Could you rephrase the specific action items needed, thanks!

![issue-5433](https://user-images.githubusercontent.com/6722114/193707308-cc41cada-8659-4175-9df3-8dca4db4040a.jpg)

**Comment by nicolevasilevsky** [comment]
Thanks for the clarification @kanems! I'll create the new term: dystonia, dopa-responsive, with or without hyperphenylalaninemia, autosomal recessive

I'm unsure if AR, BH4 deficient should be a child of DRD, so I will just leave it for now, unless you or someone else tells me otherwise.

![issue-5433-2](https://user-images.githubusercontent.com/6722114/195875985-a78fb24b-041c-4011-9788-2d5fc87184f1.jpg)
